docs/scalardb-supported-databases.md

```diff
@@ -6,15 +6,22 @@ Scalar DB supports the following databases and the databases that are compatible
 * DynamoDB
 * MySQL
 * PostgreSQL
+* Oracle Database
+* SQL Server
+* AWS Aurora MySQL
+* AWS Aurora PostgreSQL
 
 ## Compatibility matrix
 
-| Scalar DB Version   | Cassandra 3.11 | Cassandra 3.0    | Cassandra 2.2   | Cosmos DB | DynamoDB  | MySQL 8.0  | MySQL 5.7  | PostgreSQL 13    | PostgreSQL 12   | PostgreSQL 11  | Oracle 19.0.0.0.ru-2021-04.rur-2021-04.r1 | Oracle 12.2.0.1.ru-2021-04.rur-2021-04.r1 | Oracle 12.1.0.2.v24   |
-|:--------------------|:---------------|:-----------------|:----------------|:----------|:----------|:-----------|:-----------|:-----------------|:----------------|:---------------|-------------------------------------------|-------------------------------------------|-----------------------|
-| Scalar DB 3.1.0 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |
-| Scalar DB 3.0.0 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |
-| Scalar DB 2.4.1 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |
-| Scalar DB 2.3.0 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |
-| Scalar DB 2.2.2 |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |
-| Scalar DB 2.1.0 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |
-| Scalar DB 2.0.1 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |
+| Scalar DB Version | Cassandra 3.11 | Cassandra 3.0    | Cassandra 2.2   | Cosmos DB | DynamoDB  | MySQL 8.0  | MySQL 5.7  | PostgreSQL 14 | PostgreSQL 13 | PostgreSQL 12 | Oracle 19.0.0.0.ru-2021-04.rur-2021-04.r1 | Oracle 12.2.0.1.ru-2021-04.rur-2021-04.r1 | Oracle 12.1.0.2.v24   | SQL Server 2019 | SQL Server 2017 | SQL Server 2016 | AWS Aurora MySQL 3 (MySQL 8) |  AWS Aurora PostgreSQL 13 | AWS Aurora PostgreSQL 12 | AWS Aurora PostgreSQL 11 |
+|:------------------|:---------------|:-----------------|:----------------|:----------|:----------|:-----------|:-----------|:--------------|:--------------|:--------------|:------------------------------------------|:------------------------------------------|:----------------------|:----------------|:----------------|:----------------|:-----------------------------|:--------------------------|:-------------------------|:-------------------------|
+| Scalar DB 3.4 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |
```

Comment:
@scalar-boney Can you please add the result of the compatibility test for Scalar DB 3.5?

Reply:
Ok, We will add

Reply:
Since we already released Scalar DB 3.5, can you please do that by EOD today?

Reply:
@brfrn169 san,
Sorry for the delay.
Compatibility matrix updated with Scalar DB 3.5.